src/components/Footer/index.js

```diff
@@ -37,62 +39,109 @@ export const OfficeStyled = styled(Row)`
   overflow: hidden;
 `
 
-const Footer = () => {
-  return (
-    <GreyFooter>
-      <Padding top={{ smallPhone: 3, tablet: 4 }}>
+const DesktopGDPR = styled.div`
+  visibility: hidden;
+  position: absolute;
+  right: 0;
+  color: white;
+
+  a {
+    margin-left: 24px;
+  }
+
+  ${breakpoint('desktop')`
+    visibility: visible;
+  `}
+`
+
+const MobileGDPR = styled.div`
```

Comment:
I don't think we need to create 2 different components - 1 for desktop & 1 for mobile - since the content and functionality is exactly the same. Layout is better generally dealt with via CSS than creating 2 sets of HTML, if you can help it.

In the case of our project, I think the way that we generally handle this kind of situation is through the `Col` component.

If you wrap `<Social>` in one `Col` & `<GDPRLinks>` in another `Col`, then you can use the `width` prop on `Col` to control their width on different screen sizes.

Reply:
Yes . I also considered that option and something was not sitting right with me by doing 2 components. It makes sense to control via CSS.
I have done 1 component now and so far the mobile/tablet view.
Now I have a problem; I don't understand how to make turn a `Col` to 50% width. Can't find much info on it in the repo either. I see sometimes that we do stuff like `<Col width={[1, 1, 1, 1, 3 / 12]}>` but that's not speaking to me...

Reply:
I agree that it would be useful for us to document how to use our grid system but [the ticket for creating docs](https://trello.com/c/DrXHIMWe/31-document-how-our-grid-works-should-be-used-in-combination-with-the-other-style-libraries-we-use) has been deprioritised.

Our project's grid system is built on a package called `rebass grid` - [here's the section on responsive styles](https://github.com/rebassjs/grid#responsive-styles). Rebass grid is built on [`styled system`](https://github.com/styled-system/styled-system), which has [slightly more detailed docs](https://styled-system.com/responsive-styles) on this `width` prop.

So to summarise - you achieve 50% width by writing `width={1/2}`.
But we have an array, because this corresponds to the array of breakpoints in our `theme.js`. So the first four `1`s in the array correspond to the mobile breakpoints & the `3/12` will correspond to small tablet and upwards. If we wanted large tablet to have a different width, we would add another number to the array. And so on.

Hope that makes slightly more sense?

Reply:
It did! Nice and updated 👍 Thanks